saleor/checkout/fetch.py

```diff
@@ -64,14 +71,17 @@ def get_customer_email(self) -> str:
 
 @dataclass(frozen=True)
 class DeliveryMethodBase:
-    delivery_method: Optional[Union["ShippingMethod", "Warehouse"]] = None
+    delivery_method: Optional[Union["ShippingMethodData", "Warehouse"]] = None
     shipping_address: Optional["Address"] = None
-    order_key: str = "shipping_method"
 
     @property
     def warehouse_pk(self) -> Optional[str]:
         pass
 
+    @property
+    def delivery_method_order_field(self) -> dict:
```

Comment:
It could be convert into class constant field instead of creating new dict each time